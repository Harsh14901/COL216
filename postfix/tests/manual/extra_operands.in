58+85219832***924++--+
20-419-1-539-2++2482+*+
4147728-49*1454+*+**12--
30-4-1*957*3***31+0*
507-6-1-3-2426+15*23--*
78655+-80*3*4015+4+*-*
53*4+7*3*432720--*+30-
99287+124-+2685+--2-*-
88+2-20235*5425-6*-*++
795*159*-241+-7-31*-5*
