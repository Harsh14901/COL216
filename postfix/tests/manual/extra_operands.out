[*] Executing test case - 58+85219832***924++--+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 20-419-1-539-2++2482+*+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 4147728-49*1454+*+**12--
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 30-4-1*957*3***31+0*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 507-6-1-3-2426+15*23--*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 78655+-80*3*4015+4+*-*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 53*4+7*3*432720--*+30-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 99287+124-+2685+--2-*-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 88+2-20235*5425-6*-*++
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 795*159*-241+-7-31*-5*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
